From 2b45c36f062bc5e3d902d9eb7691ba9a421e4ce4 Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Tue, 21 Aug 2012 15:22:22 +1000
Subject: [PATCH synaptics] Undefine HAVE_SMOOTH_SCROLLING

None of the RHEL6 stack has smooth scrolling support and it breaks some
use-cases that rely on button remapping. So undefine it for RHEL6.

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
---
 src/synapticsstr.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/synapticsstr.h b/src/synapticsstr.h
index fbae394..7719d72 100644
--- a/src/synapticsstr.h
+++ b/src/synapticsstr.h
@@ -47,6 +47,7 @@
 #ifdef AXIS_LABEL_PROP_REL_VSCROLL
 #define HAVE_SMOOTH_SCROLL
 #endif
+#undef HAVE_SMOOTH_SCROLL
 
 /******************************************************************************
  *		Definitions
-- 
1.7.11.2

